stestr/subunit_trace.py

```diff
@@ -386,7 +387,7 @@ def trace(stdin, stdout, print_failures=False, failonly=False,
     # NOTE(mtreinish): Ideally this should live in testtools streamSummary
     # this is just in place until the behavior lands there (if it ever does)
     if count_tests('status', '^success$') == 0:
-        print("\nNo tests were successful during the run")
+        print("\nNo tests were successful during the run", file=sys.stderr)
```

Comment:
This one is even more on the edge for me. I'm not sure if stderr is better here or not. In this case we ran the tests and there were tests being run, just none were successful. Normally this gets printed if you run a test suite and all the cases in the suite are skipped. Which in the normal unittest/testtools world would be a successful run. We handle this separately here because that's normally not the desired result from a test run to skip all the tests and normally indicates a failure. All the other subunit-trace output in this case goes to stdout, I'm not sure this is sufficiently different to warrant being in stderr.